pages/testpage.tsx

```diff
@@ -2,10 +2,10 @@
 import React from "react";
 import Layout from "../components/Layout/layout";
 import CandidateTestView from "../components/CandidateTestView/CandidateTestView";
-import {TestList} from "../components/CandidateTestView/Tests/TestList";
+import {CandidateTestModel} from "../Models/CandidateTestModel";
+import {testToRender} from "./index";
 
-const TestPage: NextPage = () => {
-    const testToRender = TestList[1];
+const TestPage: NextPage<CandidateTestModel> = () => {
```

Comment:
I think _strictly_ this should just be a `NextPage` for now (as we don't actually pass in a `CandidateTestModel`)

But I suspect we are about to change this to make this correct, so fine for now.